content/docs/command-reference/init.md

```diff
@@ -32,67 +32,59 @@ that are hidden from the user. This directory is automatically staged with
 
 `--subdir` must be provided to initialize DVC in a subdirectory of a Git
 repository. DVC still expects to find the Git repository (will check all
-directories up to the root to find `.git`). This options does not affect any
-config files, `.dvc` directory is created the same way as in the default mode.
-This way multiple DVC projects (including nested ones) could be initialized in a
```

Comment:
why are we removing points about nesting, about granularity?

Reply:
> about nesting

I think "multiple DVC projects can be initialized in a single Git repository" implies "including nested ones". In general I didn't remove the nesting explanation, in fact I clarified it and expanded on it quite a bit.

> about granularity

The doc mentioned "isolated and granular" everywhere. We're really just talking about one thing I think, not 2, and "isolation" seems to me like the more correct concept.